public/partials/home/sidebar.html

```diff
@@ -33,6 +33,12 @@
     <a id="usersMainMenu" ng-show="isWSTADMIN" href="/#/users" class="list-group-item">
         <i class="fa fa-user"></i> <span translate>&nbsp;Users  </span><span class="badge">{{counts.users}}</span>
     </a>
+    <a id="usersMainMenu" ng-show="isWSTADMIN" href="/#/import" class="list-group-item">
+        <i class="fa fa-download"></i> <span translate>&nbsp;Import  </span><span class="badge">{{counts.users}}</span>
+    </a>
+    <a id="usersMainMenu" ng-show="isWSTADMIN" href="/#/export" class="list-group-item">
+        <i class="fa fa-upload"></i> <span translate>&nbsp;Export  </span><span class="badge">{{counts.users}}</span>
+    </a>
```

Comment:
Too much copy/paste :-)
The badge makes no sense here

Also, please move `&nbsp;` out of the translatable text

And you're using 'fa-download' for import and 'fa-upload' for export. Shouldn't it be the opposite ?